src/login/lobby.cpp

```diff
@@ -343,7 +343,23 @@ int32 lobbydata_parse(int32 fd)
 
                     ShowInfo("lobbydata_parse: zoneid:(%u),zoneip:(%s),zoneport:(%u) for char:(%u)", ZoneID, ip2str(ntohl(ZoneIP)), ZonePort, charid);
 
-                    if (maint_config.maint_mode == 0 || gmlevel > 0)
+                    // Check the number of sessions
+                    uint16 SessionCount = 0;
```

Comment:
Please rename this to `sessionCount`, and you could move this inside your SQL `if` statement; since it isn't used anywhere else.

Reply:
Thanks! after looking again, looks like we need it outside that scope - looks like I need to instantiate it outside or nest some `if`s